.php-cs-fixer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+/**
+ * Read excluded paths from .gitignore
+ *
+ * @param string $dir
+ *
+ * @return string[]
+ */
+function getGitignorePaths(string $dir): array
+{
+    $gitignoreFile = $dir . '/.gitignore';
+    $paths         = [];
+
+    if (!file_exists($gitignoreFile)) {
+        return $paths;
+    }
+
+    $lines = file($gitignoreFile, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+    if (!empty($lines)) {
+        foreach ($lines as $line) {
+            // Ignore comments and empty lines
+            if (strpos($line, '#') === 0 || trim($line) === '') {
+                continue;
+            }
+            // Add relative paths
+            $paths[] = trim($line);
+        }
+    }
+
+    return $paths;
+}
+
+$projectDir = __DIR__;
+$finder     = Finder::create()
+    ->in($projectDir)
+    ->name('*.php');
+
+// Exclude paths from .gitignore
+$gitignorePaths = getGitignorePaths($projectDir);
+foreach ($gitignorePaths as $path) {
+    $finder->notPath($path);
+}
+
+$config = new Config();
+
+$mandatoryRules = [
+    '@PSR12'                          => true,
+    '@PER-CS2.0'                      => true,
```

Comment:
Why not only `@PER-CS2.0`?

Reply:
I thought we wanted both, but I must have misunderstood. I'll remove PSR-12.